src/main/java/seedu/foodrem/model/util/SampleDataUtil.java

```diff
@@ -0,0 +1,73 @@
+package seedu.foodrem.model.util;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.foodrem.model.FoodRem;
+import seedu.foodrem.model.ReadOnlyFoodRem;
+import seedu.foodrem.model.item.Item;
+import seedu.foodrem.model.item.ItemBoughtDate;
+import seedu.foodrem.model.item.ItemExpiryDate;
+import seedu.foodrem.model.item.ItemName;
+import seedu.foodrem.model.item.ItemQuantity;
+import seedu.foodrem.model.item.ItemUnit;
+import seedu.foodrem.model.tag.Tag;
+
+/**
+ * Contains utility methods for populating {@code AddressBook} with sample data.
+ */
+public class SampleDataUtil {
+    public static Item[] getSampleItems() {
+        return new Item[]{
+            new Item(new ItemName("Potatoes"),
+                new ItemQuantity("10"),
+                new ItemUnit("kg"),
+                new ItemBoughtDate("2022-11-11"),
+                new ItemExpiryDate("2022-11-11")),
+            new Item(new ItemName("Cucumbers"),
+                new ItemQuantity("2000"),
+                new ItemUnit("grams"),
+                new ItemBoughtDate("2022-11-11"),
+                new ItemExpiryDate("2022-11-11"))
```

Comment:
Should create sample data with different expiry date from bought date. Possibly add some items that do not have expiry dates as well.